README.md

```diff
@@ -25,7 +25,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
```

Comment:
:smile: 

I'm fine with the output graph being ASCII. I just don't want to maintain the shape of the graph in raw ASCII.